.dockerignore

```diff
@@ -1 +1,3 @@
 .git/
+server-0/
+server-1/
```

Comment:
this is kinda unrelated but was something I came across when trying to rebuild the docker images to include `YAML::XS`.